analyzer/src/testFixtures/kotlin/TestUtils.kt

```diff
@@ -122,7 +121,7 @@ fun ProjectAnalyzerResult.withInvariantIssues() =
         issues = issues.sortedBy { it.message }.map { it.copy(timestamp = Instant.EPOCH) }
     )
 
-fun Spec.analyze(
+fun analyze(
```

Comment:
I also stumbled across these recently. I assumed that the reason for limiting the function to `Spec`
was to reduce the scope and not pollute the whole namespace with this function. What do you think about this?

Reply:
It was both: I was looking for a reasonable justification to put these into `Extensions.kt` back then, without the need to introduce a new file, and though it would be a nice "trick" limit the functions to the `Spec` context, although the context is never used. But it always annoying me that the IDE is actually dimming `Spec` to hint it being unused.

Anyway, by now these are in the `testFixture` source set, and the required file name change now includes "Test" as well, so I think the scope should be clear, and name clashes are very unlikely.

Reply:
> Anyway, by now these are in the `testFixture` source set, and the required file name change now includes "Test" as well, so I think the scope should be clear, and name clashes are very unlikely.

Agree.